config/locales/en.yml

```diff
@@ -232,9 +232,9 @@ en:
       order_devices_now: Order devices now
     invite_school_user:
       title: Invite a new user
-      will_they_order_devices: Will they place orders for devices?
-      will_they_order_devices_hint: No more than 3 users will be allowed to place orders.
-      can_order_devices: Yes, give them access to the TechSource website for ordering and a Support Portal for technical help
+      will_they_order_devices: Do you want to give them access to the Support Portal?
+      will_they_order_devices_hint: Up to 3 people can have access to the Support Portal.
```

Comment:
Can we remove 'have' from 'Up to 3 people can have access to the Support Portal'?